projects/valtimo/task/src/lib/components/task-detail-intermediate-save/task-detail-intermediate-save.component.ts

```diff
@@ -70,7 +57,9 @@ export class TaskDetailIntermediateSaveComponent {
       title: value.name,
       subtitle: `${this.translateService.instant('taskDetail.taskCreated')} ${value.created}`,
     });
-    this.getCurrentProgress(value);
+    if (this.formFlowInstanceId$.value === undefined) {
```

Comment:
Add this logic to the subscribe above, using the BehaviorSubject value like this is not a good practice